From 29cbdb5f392776ab9e1bc5c9ebc343d4b0880d1c Mon Sep 17 00:00:00 2001
From: Chris Dearman <chris.dearman@imgtec.com>
Date: Wed, 9 Dec 2015 14:40:18 -0800
Subject: [PATCH 119/153] pistachio_marduk.dts: Set sdhost bus width to 4

The drivers manage to negotiate an appropriate data width for regular
SDcard but hangs when accessing the onboard eMMC device

Change-Id: Ib57306a3d994221242ea3136f4872aab94de9dc5
---
 arch/mips/boot/dts/pistachio/pistachio_marduk.dts | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/mips/boot/dts/pistachio/pistachio_marduk.dts b/arch/mips/boot/dts/pistachio/pistachio_marduk.dts
index 78cb138..0e644a6 100644
--- a/arch/mips/boot/dts/pistachio/pistachio_marduk.dts
+++ b/arch/mips/boot/dts/pistachio/pistachio_marduk.dts
@@ -254,6 +254,7 @@
 
 &sdhost {
 	status = "okay";
+	bus-width = <4>;
 	disable-wp;
 };
 
-- 
1.9.1

